cluster/charts/crossplane/templates/deployment.yaml

```diff
@@ -134,14 +134,14 @@ spec:
             port: readyz
         ports:
         - name: readyz
-          containerPort: 8081
+          containerPort: {{ .Values.readiness.port | default "8081" }}
```

Comment:
K8s spec expects this containerPort to be of Integer type: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.31/#containerport-v1-core

unquote the default as it was before, also may be test helm template rendering   if `.Values.readiness.port | int`  is required ?